docs/json-specification.rst

```diff
@@ -0,0 +1,67 @@
+************************************
+Specification of ``signatures.json``
+************************************
+
+Goal
+####
+
+This file should be the data which is used by a program to detect with a general algorithm which operating system an iso
+image is handed to the program. This specification should not limit itself to one operating system but if this is not
+possible then we shall focus on Linux and it's Distributions.
+
+File structure
+##############
+
+The file should be a single JSON Object which has a single key with the name `breeds`. This key should contain a single
+sub-object with key-value pairs for each operating system group.
+
+Currently the following operating system groups are existing:
+
+- redhat
+- debian
+- ubuntu
+- suse
+- vmware
+- freebsd
+- xen
+- unix
+- windows
+- powerkvm
+- generic
+
+This list should be modified when new groups are added, removed or changed when changed in the JSON.
+
+Each of these operating system groups has a key for each version of it. The key should be in lowercase and should have
+a version suffix. The name of the key should be unique across the distribution section.
+
+The Key contains an object which has the following keys (type included):
```

Comment:
```suggestion
a version suffix. The name of the key should be unique across the distribution section and
contains an object which has the following keys (type included):
```
Maybe this could be simplified even more. The object in the key which has keys itself is a little bit confusing.